tests/unit/Cache/Backend/RedisCest.php

```diff
@@ -413,7 +492,7 @@ public function queryKeysWithStatsKeyAndPrefix(UnitTester $I)
         $keys = $cache->queryKeys();
         sort($keys);
 
-        $I->assertEquals(['a', 'b'], $keys);
+        $I->assertEquals(['phalcon-a', 'phalcon-b'], $keys);
```

Comment:
Is the business logic was changed? Previously we'd expect here result w/o prefix. Should we notice about this in the blogpost to alert users?

Reply:
No the business logic remains the same. The test was incorrect. The logic is:

```
$cache->get('a'); // 'a' (no statsKey, no prefix)
$cache->get('a'); // 'phalcon_a' (no statsKey, prefix "phalcon_")
$cache->get('a'); // '_PHCRphalcon_a' (statsKey "_PHCR", prefix "phalcon_")
```